app/forms/declaration/TransportInformation.scala

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.declaration
+
+import forms.MetadataPropertiesConvertable
+import play.api.data.Forms.{boolean, optional, text}
+import play.api.data.{Form, Forms}
+import play.api.libs.json.Json
+import services.Countries.allCountries
+import utils.validators.forms.FieldValidator._
+
+case class TransportInformation(
```

Comment:
Do we need need this class still?

Reply:
waiting for summary page updates

Reply:
leaving it to look into how we can handle summary page